src/doghouse/core/services/recorder_service.py

```
@@ -0,0 +1,74 @@
+                    severity=b.severity if b.severity.value > existing.severity.value else existing.severity,
+                    is_primary=b.is_primary or existing.is_primary,
+                    metadata={**existing.metadata, **b.metadata}
+                )
+            else:
+                blocker_map[b.id] = b
+
+        blockers = list(blocker_map.values())
+        metadata = self.github.get_pr_metadata(pr_id)
+
+        current_snapshot = Snapshot(
+            timestamp=datetime.datetime.now(),
```

Ruff (0.15.7)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)